solidity/interfaces/IOngoingAirdrops.sol

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+/**
+ * @title A contract that will enable users to claim from multiple ongoing airdrops, and each airdorp will have a deadline.
+ * @notice This contract will enable multiple ongoing and updateable airdrops (called campaigns). Each of this campaigns
+ *         will have an updateable deadline, and will be able to hold airdrop information for multiple tokens.
+ *         Additionally, governor will be able to shutdown a campaign whenever it choses to. Since this is a pretty
+ *         permissioned contract, governor has full control over funds.
+ */
+
+interface IOngoingAirdrops {
+  /// @notice Token and amount information
+  /// @dev Struct that will hold an ERC20 token, and an amount.
+  struct TokenAmount {
+    IERC20 token;
+    uint256 amount;
+  }
+
+  /// @notice Thrown when a campaign is invalid, or zero bytes.
+  error InvalidCampaign();
+
+  /// @notice Thrown when a campaign does not exist or merkle root is invalid.
+  error InvalidMerkleRoot();
+
+  /// @notice Thrown when trying to update a campaign or claiming with invalid token amount.
+  error InvalidTokenAmount();
+
+  /// @notice Thrown when trying to do operations on a zero address.
+  error ZeroAddress();
+
+  /// @notice Thrown when validating an airdrop claim with the proof is invalid.
+  error InvalidProof();
+
+  /// @notice Thrown when trying to update a campaign's deadline to an invalid moment.
+  error InvalidDeadline();
+
+  /// @notice Thrown when trying to claim an expired campaign.
+  error ExpiredCampaing();
+
+  /// @notice Thrown when user tries to claim a campaign without having a new claim.
+  error AlreadyClaimed();
+
+  /// @notice Emitted when a campaign is updated
+  /// @param campaign Campaign name updated
+  /// @param root Merkle root that will be used to validate campaign claims
+  /// @param tokensAllocation Array of the sum of all tokens and amounts airdropped in the campaign
+  /// @param deadline Timestamp until when the campaign is claimable
+  event CampaignUpdated(bytes32 campaign, bytes32 root, TokenAmount[] tokensAllocation, uint32 deadline);
```

Comment:
Is there a way to silence this warnings below? I mean, we don't want our events to start with `_`